### ISSUE TITLE
Image not valid

### ISSUE THREAD

**Issue by joechancellor** [MicrosoftDocs/mslearn-improve-app-scalability-resiliency-with-load-balancer#10]
https://github.com/MicrosoftDocs/mslearn-improve-app-scalability-resiliency-with-load-balancer/blob/ba950eba5260b503e66639fa5a028850ac66b7a0/create-high-availability-vm-with-sets.sh#L65

https://github.com/MicrosoftDocs/mslearn-improve-app-scalability-resiliency-with-load-balancer/blob/ba950eba5260b503e66639fa5a028850ac66b7a0/create-high-availability-vm-with-zones.sh#L122

Getting an error on this image.  Changed to Ubuntu2204 with success:

`------------------------------------------
Creating webVM1
Invalid image "UbuntuLTS". Use a valid image URN, custom image name, custom image id, VHD blob URI, or pick an image from ['CentOS85Gen2', 'Debian11', 'FlatcarLinuxFreeGen2', 'OpenSuseLeap154Gen2', 'RHELRaw8LVMGen2', 'SuseSles15SP3', 'Ubuntu2204', 'Win2022Datacenter', 'Win2022AzureEditionCore', 'Win2019Datacenter', 'Win2016Datacenter', 'Win2012R2Datacenter', 'Win2012Datacenter', 'Win2008R2SP1'].
See vm create -h for more information on specifying an image.
------------------------------------------
Creating webVM2
Invalid image "UbuntuLTS". Use a valid image URN, custom image name, custom image id, VHD blob URI, or pick an image from ['CentOS85Gen2', 'Debian11', 'FlatcarLinuxFreeGen2', 'OpenSuseLeap154Gen2', 'RHELRaw8LVMGen2', 'SuseSles15SP3', 'Ubuntu2204', 'Win2022Datacenter', 'Win2022AzureEditionCore', 'Win2019Datacenter', 'Win2016Datacenter', 'Win2012R2Datacenter', 'Win2012Datacenter', 'Win2008R2SP1'].
See vm create -h for more information on specifying an image.
--------------------------------------------------------`

**Comment by davidzmsg** [comment]
Please try using Ubuntu2204LTS to replace UbuntuLTS in create-high-availability-vm-with-sets.sh.

**Comment by davidzmsg** [comment]
I don't think it has to be an LTS version. Try replacing UbuntuLTS with Ubuntu2204 in create-high-availability-vm-with-sets.sh.

**Comment by joechancellor** [comment]
Replacing UbuntuLTS with Ubuntu2204 was successful.  I did have to change it in both files "create-high-availability-vm-with-sets.sh" and "create-high-availability-vm-with-zones.sh"
